### PR TITLE
Add pagination functionality

### PR DESCRIPTION
**Pull request by LevinGeorge**

### CODE REVIEW

File: app/models/shop.rb [hunk 1]
```diff
@@ -13,6 +13,8 @@ class Shop < ActiveRecord::Base
   mount_uploader :profile_image, UserProfileUploader
   mount_uploader :header_image, UserHeaderImageUploader
 
+  paginates_per 1
```
Comment (danielpaul) [inline app/models/shop.rb]: This should be probably `10`

File: app/controllers/products_controller.rb [hunk 2]
```diff
@@ -8,7 +8,7 @@ class ProductsController < ApplicationController
   add_breadcrumb "Products", :shop_products_path
 
   def index
-    @products = Product.all
+    @products = Product.all.page params[:page]
```
Comment (danielpaul) [inline app/controllers/products_controller.rb:11]: You have added to the products view here, but haven't included it in the view code?

File: app/models/shop.rb (hunk 1 above)
Comment (LevinGeorge) [inline app/models/shop.rb]: :+1:

File: app/controllers/products_controller.rb (hunk 2 above)
Comment (LevinGeorge) [inline app/controllers/products_controller.rb:11]: :+1:

**Pull request merged**